nesta/core/routines/arxiv/arxiv_collect_iterative_task.py

```diff
@@ -54,15 +54,18 @@ def run(self):
         database = 'dev' if self.test else 'production'
         logging.warning(f"Using {database} database")
         self.engine = get_mysql_engine(self.db_config_env, 'mysqldb', database)
+        Base.metadata.create_all(self.engine)
```

Comment:
this needs to be here in the case that you have `DROP`'ped the corresponding tables in `MySQL` to force a clean recollection - i.e. this rebuilds the tables